src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -312,12 +312,9 @@ def set_location(cmd, sku, location):
     return loc.replace(" ", "").lower()
 
 
-# check if the RG value to use already exists and follows the OS requirements or new RG to be created
-def should_create_new_rg(cmd, rg_name, is_linux):
-    if (_check_resource_group_exists(cmd, rg_name) and
-            _check_resource_group_supports_os(cmd, rg_name, is_linux)):
```

Comment:
Can we remove the helper method code as well - if this is not used anywhere else?

Reply:
Just removed the function